src/test/resources/multiple_generics.json

```diff
@@ -0,0 +1,26 @@
+{
+   "type" : "record",
+   "name" : "WrapperTest",
+   "namespace" : "com.github.avrokotlin.avro4k.schema.MultipleGenericsSchemaTest",
+   "fields" : [ {
+      "name" : "a",
+      "type" : {
+         "type" : "record",
+         "name" : "GenericValue",
+         "fields" : [ {
+            "name" : "z",
+            "type" : "string"
+         } ]
+      }
+   }, {
+      "name" : "b",
+      "type" : {
+         "type" : "record",
+         "name" : "GenericValue",
+         "fields" : [ {
+            "name" : "z",
+            "type" : "double"
```

Comment:
Is avro spec handling 2 record types with the same name but different fields ? Sounds weird to me 🤔